libs/data-mapper/src/lib/components/configPanel/AddOrUpdateSchemaView.tsx

```diff
@@ -135,21 +86,8 @@ export const AddOrUpdateSchemaView = ({
     [setUploadType]
   );
 
-  const onSelectSchemaFile = (event: React.FormEvent<HTMLInputElement>) => {
-    if (!event?.currentTarget?.files) {
-      console.error('Files array is empty');
-      return;
-    }
-
-    const schemaFile = event.currentTarget.files[0] as FileWithVsCodePath;
-    if (!schemaFile.path) {
-      console.log('Path property is missing from file (should only occur in browser/standalone)');
-    } else if (!schemaType) {
-      console.error('Missing schemaType');
-    } else {
-      setSelectedSchemaFile({ name: schemaFile.name, path: schemaFile.path, type: schemaType });
-    }
-  };
+  // eslint-disable-next-line no-param-reassign
+  errorMessage = 'this is an error';
```

Comment:
Maybe make this look a little more official, capitalization, etc?

Reply:
this was for texting, removed